service/src/main/java/skills/services/admin/UserAchievementExpirationService.groovy

```diff
@@ -47,6 +51,12 @@ class UserAchievementExpirationService {
     @Autowired
     ExpiredUserAchievementRepo expiredUserAchievementRepo
 
+    TableResult findAllExpiredAchievements(String projectId, String userId, String skillName, PageRequest pageRequest) {
+        List<ExpiredSkillRes> results = expiredUserAchievementRepo.findAllExpiredAchievements(projectId, userId, skillName, pageRequest)
+        def totalExpirations = expiredUserAchievementRepo.countExpiredAchievements(projectId, userId, skillName)
```

Comment:
you can retrieve both count and page using `Page` interface , here is an example from: `skills/services/userActions/UserActionsHistoryService.groovy:96`

```
Page<UserActionsHistoryRepo.UserActionsPreview> userActionsPreviewFromDB = userActionsHistoryRepo.getActions(
                projectIdFilter, itemFilter, userFilter, quizFilter, itemIdFilter, actionFilter, pageRequest)
        Long totalRows = userActionsPreviewFromDB.getTotalElements()
       List<DashboardUserActionRes> actionResList = userActionsPreviewFromDB.getContent()....
```